packages/@lwc/synthetic-shadow/src/polyfills/event-listener/polyfill.ts

```diff
@@ -84,7 +83,10 @@ function windowRemoveEventListener(
     }
 
     const wrapperFn = getEventListenerWrapper(listener);
-    nativeWindowRemoveEventListener.call(this, type, wrapperFn || listener, optionsOrCapture);
```

Comment:
from `wrapperFn || listener` it may seem that some times `getEventListenerWrapper` returns a falsy value.

is it fine to call removeEventListener with undefined/falsy as the listener?

Reply:
yes. It is fine. https://dom.spec.whatwg.org/#add-an-event-listener

Working example.
https://jsbin.com/locasik/edit?html,js,console